keyboards/han60/info.json

```diff
@@ -0,0 +1,355 @@
+{
+    "keyboard_name": "han60",
+    "maintainer": "qmk",
+    "url": "hanboards.com",
+    "layouts": 
+	{
+        "LAYOUT_all": 

```

Comment:
None of the layouts defined here match the content of `han60.h`

Reply:
The `layout_all` should match the format of the `.h` file? Or do I need to change `han60.h` to correspond to each layout listed in info.json?